src/webviews/apps/settings/partials/dates.html

```diff
@@ -69,17 +69,23 @@ <h2>Dates & Times</h2>
 						</span>
 					</div>
 
-					<div class="setting">
-						<div class="setting__input">
+					<div class="setting" data-enablement="currentLine.enabled">
+						<div class="setting__input setting__input--format">
 							<label for="defaultDateFormat">Date&nbsp;format</label>
+
 							<input
 								id="defaultDateFormat"
 								name="defaultDateFormat"
 								type="text"
 								placeholder="defaults to MMMM Do, YYYY h:mma"
 								data-setting
 								data-setting-preview
+								data-popup-trigger
+								disabled
 							/>
+							<label for="defaultDateFormat" title="See some sugested date/time formats">
```

Comment:
Typo: suggested

Reply:
done https://github.com/gitkraken/vscode-gitlens/pull/2623/commits/b69157a659faa1ddac7340c8a257d9f0fd3d3b34

Reply:
(Good day) Seems like this branch isn't up to date!

Reply:
Yup, that's why I started to work on my own branch with these commits